src/libraries/System.Text.Json/tests/Serialization/Stream.WriteTests.cs

```diff
@@ -356,25 +358,31 @@ public static async Task DeepNestedJsonFileTest(int depthFactor, bool ignoreNull
 
         [Theory]
         [InlineData(1)]
-        [InlineData(16)]
-        public static async Task DeepNestedJsonFileCircularDependencyTest(int depthFactor)
+        [InlineData(4)]
+        public static async Task NestedJsonFileCircularDependencyTest(int depthFactor)
```

Comment:
Why are we changing this test? Especially the depth factor and length?

Reply:
Discussed offline